File: documentation/user/en/query/ordering/examples/reference/reference-attribute-natural-multiple.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					referenceHaving(
						"groups",
						entityHaving(
							attributeInSet("code", "sale", "new")
						)
					)
				),
				orderBy(
					referenceProperty(
						"groups",
						attributeNatural("orderInGroup", ASC)
					)
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContentWithAttributes(
							"groups",
							attributeContent("orderInGroup")
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/reference/reference-attribute-natural-multiple.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		ReferenceHaving(
        			"groups",
        			EntityHaving(
        				AttributeInSet("code", "sale", "new")
        			)
        		)
        	),
        	OrderBy(
        		ReferenceProperty(
        			"groups",
        			AttributeNatural("orderInGroup", Asc)
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContentWithAttributes(
        				"groups",
        				AttributeContent("orderInGroup")
        			)
        		)
        	)
        )
	)
);